### ISSUE TITLE
Support Multiple Octoprint Systems

### ISSUE THREAD

**Issue by MatthewTheRaven** [2ndalpha/octoremote#3]
Is there any way to connect to a different server after configuring for one?

I have two 3d printers, each with its own Octoprint connection. Ideally I would be able to save multiple configurations and switch on the fly, but at the very least, it seems like I can't disconnect from one and connect to the other without deleting the app altogether. Am I missing an option somewhere?

**Comment by richgain** [comment]
+1 for me. 
I'm having exactly the same issue.

**Comment by spatz0r** [comment]
+2 for me also.  I have 2 printers with 2 octoprints.  Being able to monitor and control both would be great!

**Comment by 2ndalpha** [comment]
It does make sense but currently Octo Remote supports only one printer at the time.
I'm planning to make it work with multiple printers but unfortunately it's not a trivial task, it will take lots of time to code it.

**Comment by MasterG403** [comment]
Is there a way to have a prompt/menu to allow us to use multiple OctoPi device ?   Not multiple printers on single device, only a way to be able to connect to another raspberry Octopi device.  By now once we are connected to 1 Octopi, we are unable to disconnect from it and connect to another one.

**Comment by John-Mc** [comment]
Maybe I'm misunderstanding, but if you give your second Raspberry a different name (say OctoPi2), can't you connect to that without signing out of your first one?

**Comment by MasterG403** [comment]
Yes – if you have multiple raspberry, but not if you use the same rasp for multiple printers.

**Comment by MatthewTheRaven** [comment]
The problem right now is that there doesn't seem to be any way in this app to sign out of an octopi system, regardless of whether it's hosted on a single raspberry pi or multiple. Even if the system just had a reset option to delete the current configuration, that would be a start. As far as I can tell, to do so right now you either need to delete the app or make your currently configured pi unavailable somehow.

**Comment by MasterG403** [comment]
We need to completely uninstall the app because it's linked via API code and there is no way to unlink it other than uninstalling and reinstalling. 

I agree, just a simple "reset" button will be a good start..   But in the end, having to choose which raspberry we want to connect to will be a tremendous time saver.

**Comment by 2ndalpha** [comment]
Next version will have access to settings from the "burger" menu on the right.
Unfortunately there is still long way to go to truly support multiple printers.
